virttest/qemu_devices/qcontainer.py

```diff
@@ -2030,10 +2031,14 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
         :param bus: the busid of parent bus(optional, virtio only)
         :param nr: the nr(port) of the parent bus(optional, virtio only)
         :param reg: reg option of isa-serial(optional)
+        :param bus_extra_params: extra virtio_serial_pci params
         :return: the device list that construct the serial device
         """
 
         devices = []
+        if bus_extra_params:
+            bus_params = (dict([_.split('=')
+                                for _ in bus_extra_params.split(',') if _]))
```

Comment:
An extra brackets although the output is same. It is up to you

Reply:
To avoid the newline character "\" I think we should keep this.